src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs

```diff
@@ -20,6 +20,10 @@ namespace System.Text
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public readonly struct Rune : IComparable<Rune>, IEquatable<Rune>
     {
+        private const char HIGH_SURROGATE_START = '\ud800';
+        private const char LOW_SURROGATE_START = '\udc00';
+        private const int HIGH_SURROGATE_RANGE = 0x3FF;
```

Comment:
Can we use PascalCase for such consts?

Reply:
I can. I copied this from existing code in `CharUnicodeInfo`. Do you think I should change all the existing occurrences to PascalCase as well?

https://github.com/dotnet/runtime/search?l=C%23&q=HIGH_SURROGATE_START